lib/identity_cache/configuration_dsl.rb

```diff
@@ -315,6 +315,9 @@ def ensure_cacheable_association(association, options)
         options[:inverse_name] ||= self.name.underscore.to_sym
         child_class = association_reflection.klass
         raise InverseAssociationError unless child_class.reflect_on_association(options[:inverse_name])
+        unless options[:embed] == true || child_class.include?(IdentityCache)
+          raise UnsupportedAssociationError, "associated class must include IdentityCache to be cached without full embedding"
```

Comment:
Maybe include the name of the associated class in the error message? For extra clarity.